site/content/docs/5.0/examples/floating-labels/floating-labels.css

```diff
@@ -43,18 +43,6 @@ body {
   transition: all .1s ease-in-out;
 }
 
-.form-label-group input::-webkit-input-placeholder {
```

Comment:
Better put these back. The prefixes are automatically added based on our browserslist config.